src/components/CalibrationManagerCells.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+    ModalBody, 
+    ModalContent,
+    ModalFooter, 
+    ModalHeader, 
+    ModalCloseButton,
+    Button,
+    Th,
+    Td,
+    Link,
+} from "@chakra-ui/react";
+import { MinUser } from "shared/User";
+import trpc from "trpc";
+import ModalWithBackdrop from "./ModalWithBackdrop";
+import UserSelector from "./UserSelector";
+import { useState } from "react";
+import { formatUserName } from "shared/strings";
+import { MdEdit } from 'react-icons/md';
+import { Calibration as SharedCalibration } from 'shared/Calibration';
+
+export function CalibrationManagerHeaderCells() {
+    return <>
+        <Th>面试主管</Th>
+    </>;
+}
+
+export function CalibrationManagerCells({ calibration, refetch }: {
+    calibration: SharedCalibration,
+    refetch: () => void,
+}) {
+    const [ editing, setEditing ] = useState<boolean>(false);
+    return <>
+      {editing && <ManagerEditor calibration={calibration} 
+        manager={calibration.manager}
+        onClose={() => setEditing(false)} refetch={refetch} />}
+
+      <Td><Link onClick={() => setEditing(true)}>
+        {calibration.manager ?
+           formatUserName(calibration.manager.name)
+        :
+        <MdEdit />
+        }
+      </Link></Td>
+    </>;
+}
+
+function ManagerEditor({ calibration, manager, refetch, onClose } : {
+    calibration: SharedCalibration,
+    manager: MinUser | null,
+    refetch: () => void,
+    onClose: () => void,
+ }) {
+    const saveManager = async (managerIds: string[]) => {
+        if (managerIds.length == 0) return;
```

Comment:
in this case should it save the managerId as null?

Reply:
See previous examples:
https://github.com/yuanjian-org/app/blob/a5036edb872d2bf9b7d5f352cce480ddc97c3317/src/components/pointOfContactCells.tsx#L72

https://github.com/yuanjian-org/app/blob/a5036edb872d2bf9b7d5f352cce480ddc97c3317/src/pages/mentees.tsx#L376

Reply:
these are all bugs and not WAI and should be updated, too